inception-kb/src/main/java/de/tudarmstadt/ukp/inception/kb/graph/KBConcept.java

```diff
@@ -53,8 +53,8 @@
     /* Commented out until the functionality which uses them is actually implemented
     static {
         ValueFactory factory = SimpleValueFactory.getInstance();
-        CLOSED = factory.createIRI(KnowledgeBaseService.INCEPTION_SCHEMA_NAMESPACE, "closed");
-        ABSTRACT = factory.createIRI(KnowledgeBaseService.INCEPTION_SCHEMA_NAMESPACE, "abstract");
+        CLOSED = factory.createIRI(IriConstants.INCEPTION_SCHEMA_NAMESPACE, "closed");
+        ABSTRACT = factory.createIRI(IriConstants.INCEPTION_SCHEMA_NAMESPACE, "abstract");
```

Comment:
Do we still use these?

Reply:
Like the comments say: not right now. I'd keep it the way it is (commented out).